gen.sh

```diff
@@ -1,67 +1,30 @@
 #!/bin/bash
-# Script to generate the Protobuf classes from googleapis/google-cloudevents
 set -e
 
-echo "~ START"
-PROTOBUF_VERSION=3.12.3
+EVENTS_SPEC_REPO='https://github.com/michaelawyu/google-cloudevents'
```

Comment:
Can we please create a PR for the non-fork version of the spec repo before continuing with this PR?

I don't want to create a dependency on this fork.